update-docker-tags.go

```diff
@@ -17,8 +17,8 @@ import (
 	"github.com/pkg/errors"
 )
 
-// TODO: make this not Sourcegraph-specific
 var TAG_PATTERN = regexp.MustCompile(`(sourcegraph/.+):(.+)@(sha256:[[:alnum:]]+)`)
+var GENERIC_TAG_PATTERN = regexp.MustCompile(` (\S+.+):(.+)@(sha256:[[:alnum:]]+)`) //https://regex101.com/r/hP8bK1/37
```

Comment:
```suggestion
var GENERIC_TAG_PATTERN = regexp.MustCompile(`\b(\S+.+):(.+)@(sha256:[[:alnum:]]+)`) // https://regex101.com/r/hP8bK1/37
```

Would this be better?

Reply:
also rename the var to just `PATTERN`, see my other comment here: https://github.com/slimsag/update-docker-tags/pull/4/files#r433580176